sdk/cosmosdb/cosmos/tslint.json

```diff
@@ -7,7 +7,9 @@
     "object-literal-sort-keys": false,
     "member-ordering": false, // TODO: might want to look at this eventually...
     "no-floating-promises": true,
-    "import-blacklist": [true, "assert", "util"]
```

Comment:
This is fine but just as a note, TSlint is deprecated and we are moving to ESlint. If you/the team has bandwidth it would be good to migrate now.

Reply:
I logged #3239 to track the work.